nrfcloud/calculateCosts.ts

```diff
@@ -0,0 +1,92 @@
+import { getCurrentMonth } from '../util/getCurrentMonth.js'
+
+export type LocationService = keyof LocationServices
+
+export type LocationServices = {
+	AGPS: {
+		requests: number
+		devicesUsing: number
+	}
+	PGPS: {
+		requests: number
+		devicesUsing: number
+	}
+	SCELL: {
+		requests: number
+		devicesUsing: number
+	}
+	MCELL: {
+		requests: number
+		devicesUsing: number
+	}
+	WIFI: {
+		requests: number
+		devicesUsing: number
+	}
+}
+
+export type UsageSummary = {
+	currentDevices: {
+		total: number
+		bluetoothLE: number
+		gateway: number
+		generic: number
+	}
+	services: {
+		date: string
+		fotaJobExecutions: number
+		storedDeviceMessages: number
+		locationServices: LocationServices
+	}[]
+}
+
+export type Prices = {
+	currentDevices: number
+	storedDeviceMessages: number
+	fotaJobExecutions: number
+	AGPS: number
+	PGPS: number
+	SCELL: number
+	MCELL: number
+	WIFI: number
+}
+
+export type ServicePrice = keyof Prices
+
+export const nRFCloudPrices: Prices = {
+	currentDevices: 0.1,
+	storedDeviceMessages: 0.0001,
+	fotaJobExecutions: 0.1,
+	AGPS: 0.001,
+	PGPS: 0.001,
+	SCELL: 0.001,
+	MCELL: 0.002,
+	WIFI: 0.002,
+}
+
+export const calculateCosts = async (
```

Comment:
```suggestion
export const calculateCosts = (
```

This function does not need to be async.